src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -291,12 +294,13 @@ def parse_performance_features(log_lines, strategies, arguments):
 
   # new_cov_* is a reliable metric when corpus subset strategy is not used.
```

Comment:
Should we record these for all the strategies? Even without the subset it's not super reliable, as it doesn't account for corpus merging, i.e. it's affected by the unstable coverage. Plus, we can always filter out irrelevant runs using BigQuery, so we don't need to dismiss the stats based on the strategy, we should upload everything.

Reply:
just makes data studio stuff more complicated. see also reason below.

Reply:
I'm afraid it's inevitable that we need to filter out certain values for different columns. It's just a price we have to pay for having so many stats and configurations :)

Reply:
Chatted offline, the final argument was that we surface these values in Fuzzer Stats, and we don't filter by strategy there, which creates very confusing picture if we record the values from runs with corpus subset strategy.